app/assets/javascripts/components/select.es6.jsx

```diff
@@ -1,4 +1,15 @@
 class Select extends React.Component {
+  constructor(props) {
```

Comment:
The constructor there is pulling in the props to set state for the component, which is how the value gets set. We might be able to do it a different way, but this is the standard way to handle select components according to the React docs.